src/common/product_models/heatmap_common_models.py

```diff
@@ -52,6 +53,26 @@ class ColumnType(str, Enum):
     """ A boolean. """
 
 
+def trans_column_type_heatmap_to_attribs(col_type: ColumnType) -> AttribsColumnType:
+    """
+    Translate a heatmap column type to an attributes column type.
+    This method is suitable for models like filters that exclusively operate on attributes column types
+
+    col_type: the heatmap column type
+    """
+    heatmap_to_attribs_mapping = {
```

Comment:
I would make this a module level constant

Reply:
👍

Reply:
:+1: